### ISSUE TITLE
Well it's a context menu so whatever your context menu key is on your keyboard. There's a dedicated Windows key for that.

### ISSUE THREAD

**Issue by davegi** [RickStrahl/MarkdownMonster#764]
Well it's a context menu so whatever your context menu key is on your keyboard. There's a dedicated Windows key for that.

For example, on my Dell XPS Laptop that key is `fn-Ctrl`. On my main keyboard that key is on the number pad by the scroll lock. Your keyboard may vary or have some other key combo for that but usually it's marked on the keyboard as a square with lines in it (ie. the dropdown).

_Originally posted by @RickStrahl in https://github.com/RickStrahl/MarkdownMonster/issues/762#issuecomment-799793334_

**Comment by davegi** [comment]
What I'm after (out of the box) is for MM to allow me to invoke the spell-checker on a misspelled word (one's with the micro-dotted, red underline) w/o taking my hands off the keyboard. My current keyboard (Steelseries) does not have a menu/hamburger key, and yes, I can always map a key. However. my experience is that many apps, use `<SHIFT>-<F10>` to invoke the identical menu that would be invoked via right-clicking the mouse, in the current location of the cursor - which is what I expected of MM. The contents of this menu vary (often by adding menu items), depending on context, _e.g._, the menu displayed for a correctly spelled word and an incorrectly spelled word would be different w/the latter being some merge of the spell-check menu and the _normal_ menu.

Bottom line, I don't mean to be demeaning in my explanation - just thorough. I really enjoy using MM and trust that you know what I'm trying to describe but having to take my hands off the keyboard to spell check/correct a word really slows things down. 

Is there an out of the box solution or do I need to resort to binding a keystroke? Thanks.

**Comment by RickStrahl** [comment]
What editor does? Neither Visual Studio, nor VS Code open context menus on `Shift-F10` for me. 

My point is Windows has a key for this and most keyboards have some (function?) key combination that access that key - so why duplicate it? For most laptops or compact keyboards that key is `fn-rightctl`.  If your keyboard doesn't have it, then that's really an issue with your hardware as Windows has explicit support for this. The key is probably there but it's just not marked on your keyboard.

I agree in that I wish the key was more universal and placed consistently, but I'm not going to map another key if there is support for it natively.
